modules/rest-api/pages/rest-bucket-create.adoc

```diff
@@ -1099,6 +1099,15 @@ If the bucket cannot created due to a missing or incorrect parameter, a `400` re
 If the URL is incorrectly specified a `404 (Object Not Found)` error is returned.
 Failure to authenticate gives `401 Unauthorized`.
 
+[#notes]
+== Notes
+
+Couchbase Server 7.6.2 added a property named `enableCrossClusterVersioning` to support a future feature. 
+Do not change the value of this property.
+
+CAUTION: Once enabled, you cannot turn off the `enableCrossClusterVersioning` property. 
+The only way for you to undo setting this value to `true` is to backup your data, create a new database, and reload the data.
```

Comment:
Should be "create a new bucket", not "create a new database".
The only way for you to undo setting this value to `true` is to backup your data, create a new bucket, and reload the data.